systemtest/apmservertest/config.go

```diff
@@ -38,6 +38,7 @@ const (
 	defaultKibanaPort = "5601"
 	defaultKibanaUser = "apm_user_ro"
 	defaultKibanaPass = "changeme"
+	defaultKibanaPath = ""
```

Comment:
Base path maybe? To me, "path" sounds like it's referring to a filesystem path to where Kibana is located.

Reply:
changed to `defaultKibanaBasePath`